spec/unit/webrtc/callFeed.spec.ts

```diff
@@ -85,5 +85,129 @@ describe("CallFeed", () => {
                 expect(feed.isVideoMuted()).toBeTruthy();
             });
         });
+
+        describe("voice activity detection", () => {
+            it("voice activity should disable audio track", () => {
+                feed.stream.addTrack(
+                    //@ts-ignore Mock
+                    new MockMediaStreamTrack("track", "audio", true),
+                );
+
+                feed.setVoiceActivityThreshold(Infinity);
+
+                setTimeout(() => {
+                    expect(feed.stream.getAudioTracks()[0].enabled).toBe(false);
+                }, 1000);
+            });
+
+            it("voice activity should enable audio track", () => {
+                feed.stream.addTrack(
+                    //@ts-ignore Mock
+                    new MockMediaStreamTrack("track", "audio", true),
+                );
+
+                feed.setVoiceActivityThreshold(-Infinity);
+                setTimeout(() => {
+                    expect(feed.stream.getAudioTracks()[0].enabled).toBe(true);
+                }, 1000);
+            });
```

Comment:
Do these tests test something specific that is not tested by the two tests below?

Reply:
Are they testing if `VADEnabled === true` ? Could that be more explicit?

Reply:
They would be the baseline, aka volume is higher than threshold therefore send, the new ones are specifically made for the cooldown, but they of course will also use vad.
We can also remove these if you want that, since technically they do the same, the ones below just add another step.